server/controllers/finance/trialBalance/index.js

```diff
@@ -97,3 +97,20 @@ exports.postToGeneralLedger = function postToGeneralLedger(req, res, next) {
     .catch(next)
     .done();
 };
+
+exports.unpostTransactions = (req, res, next) => {
+
+  const { recordUuids } = req.body;
```

Comment:
Because this is a such a super user privilege, let's check the user privilege again here.  Look up the user ID and see if their role allows them to do this and send a HTTP error 403 if not.